#                      Generic Netlink
#
# This library is free software; you can redistribute it and/or
# modify it under the terms of the GNU Lesser General Public
# License as published by the Free Software Foundation version 2.1
# of the License.
#
# Copyright (c) 2003-2008 Thomas Graf <tgraf@suug.ch>

diff --git i/Makefile w/Makefile
index 6f37160..583a060 100644
--- i/Makefile
+++ w/Makefile
@@ -1,5 +1,5 @@
 CC=gcc
-WFLAGS=-Wall
+WFLAGS=-Wall -fPIC
 CFLAGS=-O2
 INCLUDES=-Iinclude
 
